client/pwa/src/service-worker.ts

```diff
@@ -68,11 +70,14 @@ self.addEventListener("message", (e: ExtendableMessageEvent) => {
     (async () => {
       switch (e?.data?.type) {
         case "update":
-          return updateContent(self, e?.data, e);
+          unpacking = updateContent(self, e?.data, e);
```

Comment:
why have you put this here? Why not just return updateContent as before?

Reply:
That's sets the global `unpacking` promise so the `keepalive` event can return it.